sdks/python/apache_beam/io/avroio.py

```diff
@@ -105,6 +105,15 @@ def __init__(self, file_pattern=None, min_bundle_size=0, validate=True,
 
     .. testoutput::
       :hide:
+      :skipif: python_version_major == 2
```

Comment:
Is the idea to keep supporting docs generation on Python 2.7? I don't see the point as the main use is to publish docs on the Beam website.

Reply:
I assumed that dropping python 2 support for generating docs should not be done until Beam officially does.
Removed `:skipif: python_version_major == 2` from avroio.py and other files.